docs/source/user_guide/connector/connector.rst

```diff
@@ -1,86 +1,153 @@
 .. _dataconnector:
 
-=========
+===========
 Connector
-=========
+===========
 
 .. toctree::
     :maxdepth: 1
 
-    DC_DBLP_tut
+    connector_authorization
+    connector_concurrency
+    connector_pagination
     DC_Spotify_tut
```

Comment:
Shall we put the links to the example notebooks here?

Reply:
Done.